CI/.drone.yml

```diff
@@ -38,6 +38,6 @@ steps:
   # generate test scripts
   - /bin/bash bin/tests/run-all-test
   # generate all petstore samples (client, servers, doc)
```

Comment:
Are these for v2.X samples only? Can you include that in your comment on line 40?

Reply:
I don't understand how that provides clarity to the comment... "All petstore samples" means everything.

Reply:
btw something you might not be aware of is that all samples are being converted to OAS 3.

Reply:
Ah, I wasn't aware of that